### ISSUE TITLE
Recurring events while acting as_user

### ISSUE THREAD

**Issue by FineTralfazz** [WinRb/Viewpoint#200]
Is there a way to see recurring event instances while using the act_as option in EWSClient.get_folder? I know you can see them with find_items, but that doesn't seem to be compatible with act_as.

**Comment by zenchild** [comment]
Good question. Did you figure it out? I do not have an answer for you.

**Comment by FineTralfazz** [comment]
No, I didn't. It appears to be a limitation in Exchange itself. I haven't found a reasonable workaround yet.
